data-processing/entities/definitions/commands/detect_definitions.py

```diff
@@ -866,10 +872,17 @@ def process(
                     t.tex_path,
                     item.arxiv_id,
                 )
+                type_ = (
+                    "abbreviation"
+                    if t.text in abbreviations
+                    else "term"
+                    if t.text in term_phrases
+                    else "unknown"
+                )
```

Comment:
Okay, now you combined terms and abbreviations but having a type in it. I wonder how you combine these with nicknames though.

Reply:
Hey @dykang ! It's a good question. Right now, we don't extract term references for symbols, because term references for symbols are already extracted elsewhere in the pipeline (namely, the `extract_symbols.py` command).

Then, the `upload.py` file for definitions loops through all `Definiendum`s (i.e., the instances of symbols that have nicknames), scans the extracted symbol data for all other appearances of those symbols, and then uploads definition information for each of those symbol appearances. Does that make sense? Might there be a clearer way to implement behavior in the code?

Reply:
Thanks for the clarification. I think this makes sense to me!